### DISCUSSION TITLE
How to set the last page of linear flow

### DISCUSSION THREAD

**Discussion by arooni** [koreader/koreader#11702] [General]
I saw on 

https://github.com/koreader/koreader/issues/10253

There was reference to contextual menu that would let me pick the last page for percentage complete Calculations, but I couldn't seem to find it in the page browser ui.

Am I missing something here ?

I'm currently reading a book that has like a two hundred or three hundred page bibliography so it'd be really handy to eliminate those pages from the percentage complete calculation

**Comment by Frenzie** [comment]
First you have to go to ToC/bookmark settings → enabled custom hidden flows. Then you can long press on a page in the page browser.

**Comment by arooni** [comment]
Ok thank you!  That works to enable the hidden flow, but I don't see anything really changing other than the TOC entries, and a cool shading section on the page browser.  The % complete and such don't seem to have changed at all.

**Reply by jonnyl2** [reply]
Did you make sure the pages you _don't_ want to read are shaded? Just to make sure you have it set up right: If you have a 1000 page book and you want to read pages 30-700 only, you would go to page 1 and select `Start hidden flow here`, then you would go to page 30 and select `Restart regular flow here` and then go to page 701 and select `Start hidden flow here`.

**Reply by arooni** [reply]
Well I *think* I got it right.  I went to bibliography page 700, then hit "Start hidden flow here" and pages 700-1000 (bibliography I do not want to read) gets greyed out but the % and status bar calculations don't seem to get updated, even after restarting koreader.

**Reply by jonnyl2** [reply]
That's very strange. That sounds right and it should take effect immediately. Do your regular page numbers have the double slash in the status bar, e.g. 100//700 instead of 100/1000?

**Reply by arooni** [reply]
OK I understand what's happening now.  For whatever reason the *bottom status bar* is completely correct and that // bar thing is exactly what you state.

However, the *top status bar (aka the alternate status bar)* doesn't seem to be in sync with the bottom status bar, meaning that I'm much further along at any given time on the bottom status bar then the alternate status bar.

I typically read with both the bottom and the alternate status bar simultaneously.  Is there a way to get that one to update based on the hidden pages too?

**Reply by jonnyl2** [reply]
> I typically read with both the bottom and the alternate status bar simultaneously. Is there a way to get that one to update based on the hidden pages too?

I don't think that's currently possible or planned. Pinging @poire-z just in case.

**Reply by poire-z** [reply]
Not possible.
https://github.com/koreader/koreader/issues/9020#issuecomment-2045865939
> We don't love the top status bar. So you all should start to hate it too - this will help :)
